app/src/main/java/com/framgia/soundclound/data/source/local/Sqlite/AlbumLocalDataSource.java

```diff
@@ -0,0 +1,246 @@
+package com.framgia.soundclound.data.source.local.Sqlite;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import com.framgia.soundclound.data.model.Album;
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.data.source.AlbumDataSource;
+import com.framgia.soundclound.util.BindingUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.framgia.soundclound.data.source.local.Sqlite.AlbumType.ID_ALBUM;
+import static com.framgia.soundclound.data.source.local.Sqlite.AlbumType.IMAGE_ALBUM;
+import static com.framgia.soundclound.data.source.local.Sqlite.AlbumType.LIST_TRACK;
+import static com.framgia.soundclound.data.source.local.Sqlite.AlbumType.NAME_ALBUM;
+import static com.framgia.soundclound.data.source.local.Sqlite.AlbumType.NUMBER_SONG;
+import static com.framgia.soundclound.data.source.local.Sqlite.DatabseType.DATABASE_NAME;
+import static com.framgia.soundclound.data.source.local.Sqlite.TableName.TABLE_ALBUM;
+
+/**
+ * Created by Bui Danh Nam on 8/1/2018.
+ */
+
+public class AlbumLocalDataSource extends SQLiteOpenHelper implements AlbumDataSource {
+    private static int VERSION = 1;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName